docs/guides/transfer_learning_full.md

```diff
@@ -0,0 +1,370 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: 'Python 3.8.11 (''.venv'': venv)'
+  language: python
+  name: python3
+---
+
+# Transfer Learning
+
+
+### Ideas
```

Comment:
Is this meant to be part of the final version? It looks like a bit of an odd section to start with.